department-of-veterans-affairs/va.gov-cms
# VACMS-1191: New and refactored path patterns

## Description

See #1191 , #1328. 

- Added path pattern for banners.
- Deleted duplicate pattern for events.
- Added pattern for VAMC system health services & facility health service
- Normalized path patterns naming to match content types.
- Added hook_update() to regenerate aliases for select CTs. Also included `event` CT here - see #1328 

## Testing done

Visual

## QA steps

1. Review path patterns at /admin/config/search/path/patterns
1. Check if aliases are generated according to defined patterns /admin/config/search/path/patterns by  filtering Content view by each of targeted content types (Event, VAMC system banner alert with situation updates, VAMC facility health service, VAMC system health service, Publication, Support Service) 
1. View a node from each of targeted content types and verify that new revision was not created when the node was saved via hook_update
